### ISSUE TITLE
Unable to install ce7.2

### ISSUE THREAD

**Issue by RiotGamesU** [cheat-engine/cheat-engine#1514]
When I install the ce7.2 installation package downloaded from the official website, no new programs will be installed on my computer after the installation is over.
Tried to run by non-administrators and administrators, all in the above situation
Operating system: the latest version of win10
Operation steps: Open the installer, click Next, click Decline, wait for the installation to end, it shows that the installation is successful, click Finish
ps: windows defender will prompt "unwant...app...", but there is actually no program installed on my computer

**Comment by DAmNRelentless** [comment]
I have the same issue and no defender warning.

**Comment by cheat-engine** [comment]
Is internet enabled for the installer(It will download the CE files during install)  and have you checked other anti viruses that may be protecting you from installing Cheat Engine ?

**Comment by DAmNRelentless** [comment]
Internet is enabled and no anti virus is installed. Defender is disabled as far as it can be.

**Comment by cheat-engine** [comment]
Runtime protection for defender is disabled?  Also reputation based protection? (which is hidden somewhere else)

And no hardware firewall/adblocker that blocks downloads from known advertisers ?  (the CE files later downloaded are provided by a advertiser host)

**Comment by DAmNRelentless** [comment]
The download is fine. The installer is runnable and the install procedure finishes with a success message. Yet, no files are actually transferred onto the machine.
I have absolutely no antivirus and defender is disabled to the maximum even with a lot of registry tweaks. There is no way something like this is blocking the process.

It has to be something with the installer.

**Comment by RiotGamesU** [comment]
> Is internet enabled for the installer(It will download the CE files during install) and have you checked other anti viruses that may be protecting you from installing Cheat Engine ?

Yes, the network connection is available. I clicked accept a few times, and it can install opera or avast on my computer.
Supplementary information 1: Before installing ce7.2, ce6.4 was installed on my computer. When ce7.2 was installed for the first time, ce6.4 was not uninstalled (I thought it was possible to overwrite the installation). After the first installation, I found no effect, and then tried to manually uninstall ce6.4, but when I run unit000.exe (it seems to be the name), there is a pop-up prompt "unit000.dat is not exist...", I then manually deleted the entire ce6 .4 folder and the ce6.4 folder under "C:\Users\beilosuz\AppData\Roaming\Microsoft\Windows\Start Menu\Programs\", and use the Uninstall Tool to clean up ce6.4, but the installation of ce7.2 is still no effect
Supplementary information 2: When running the ce7.2 installer, if you click accept, the adware will be installed, but after the installation is over, windows defender will not prompt "unwant...app...", and if you click decline, it will not Install adware, but after installation, windows defender will prompt "unwant...app..."

**Comment by RiotGamesU** [comment]
> Runtime protection for defender is disabled? Also reputation based protection? (which is hidden somewhere else)
> 
> And no hardware firewall/adblocker that blocks downloads from known advertisers ? (the CE files later downloaded are provided by an advertiser host)

Sorry, I forgot to turn off the Real-time protection of windows defender (when I turn off Real-time protection, it will start by itself after a while), I tried to run the ce7.2 installer again, this time at the end of the installation interface "Open...when exit..." was displayed, and then ce7.2 was started. Thank you for such a good project!
ps1: I haven't used ce for a long time. Which url should be filled in the CEShare community URL of CEShare Initial Setup at the first startup?
ps2: Question about manually compiling ce7.2 1: I tried to manually compile ce7.2 in the manner of readme.md. The first compilation was Run->Build, and cheatengine-i386.exe was generated in the bin\ directory. , And the above mentioned CEShare Initial Setup and whether to run the tutorial prompts, and the choice of the program language, the default is English, and I chose Chinese, but when I exit ce7.2 and reopen it When, still in English
ps3: Question about manually compiling ce7.2 2: When I compile for the second time, I chose the first three modes of Run->Compile many Modes to compile, and cheatengine-i386.exe was generated in the bin\ directory. , cheatengine-x86_64.exe and cheatengine-x86_64-SSE4-AVX2.exe. At this time, when running any of these three programs, it is Chinese language, but when I run any of these three, click the help option The Cheat Engine Tutorial, Cheat Engine Tutorial (64-Bit) and Cheat Engine Games under the card have no response. Clicking on the other options under the help tab can all run normally

**Comment by cheat-engine** [comment]
for ceshare you can pick something from the dropdown menu or enter one from a site that hasn't registered if you know one

the tutorial projects need to be compiled seperately
